### PR TITLE
Feature/user interface

### PR DESCRIPTION
**Pull request by jpdf00**
Create the user interface:
 - Accepts name for player 1
 - Accepts name for player 2
 - Prints the board state
 - Mocks the flow of the game.

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED 🛑 

Hello team! 👋 
Great jobs far in this project! 👍 
However, we still need to implement some changes in order to get the project approved 💪 
Kindly check my comments below 👇 apply the changes and resubmit your project for a new code review ⏩ 
Good luck! 😃 
## Mandatory changes
- the game should tell which player turn it is (without computing who the winner is)
- The game should end after 9 turns
- the game should inform the player if the selected move is a winning move (without computing the winning move)
- the game should inform the player if the selected move is a draw move (without computing draw)
- the game should repeat all actions for the next player's move

**Review by GzimAsani — APPROVED** [review]
# Status: Project Approved! 🥇 
## Hello **team** Great job on implementing all the requirements from the previous TSE :+1:
### You can merge this branch into the main branch when you're ready and go on to your next project.:running::running:

Hint: _Do not forget to submit your project completion form after merging this branch._

Good luck in future projects :+1: 
Happy coding 👨‍💻

**Comment by jpdf00** [comment]
Reopening

**Pull request merged**
